permissions/Base.permission

```diff
@@ -141,6 +141,10 @@ protocol unix
 ### environment
 shell none
 
+### device info
+private-etc hw-release
+whitelist /usr/share/ssu/board-mappings.d
```

Comment:
Hmm, release and ssu stuff... not directly related to this change, but

We already allow /etc/os-release, but as that is a symlink to /etc/sailfish-release -> is it actually working? should both names be whitelisted.

Also we have those HW feature probes that modify ui behavior in select places. Probably it is just settings, but we might need allow access to /usr/share/csd/settings.d now/later.

And then there is ssu.ini that contains public consumption data like "brand" among not so public things.

Reply:
This depends on which parts of the device info API we want to allow, the qml side is quite simple but the c++ side has more features.

Reply:
I would now do the things that are useful for fixing camera and file another task for the rest. Also I think the changes here look good enough.

Reply:
Let's do so... New tasks should be created from the rest.